build.psm1

```diff
@@ -539,6 +539,12 @@ Fix steps:
         }
     }
 
+    if ($Environment.IsUbuntu)
+    {
+        # psrp 1.3* now depends on libgssapi_krb5 for Kerberos support.
+        $null = New-Item -Force -ItemType SymbolicLink -Target "/usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2" -Path "$publishPath/libgssapi_krb5.so" -ErrorAction Stop
```

Comment:
`libgssapi_krb5.so.2` is already link. It seems a right place for the fix is PSRP client repo - it should depend on `libgssapi_krb5.so.2` not `libgssapi_krb5.so`.

Reply:
At this point, I don't have a better fix...

I raised that question with the authors and for other platforms,  libgssapi_krb5.so or an explicit link is created when the component is installed. On Ubuntu, this is not the case.
The PSRP omi provider install already handles this. 

For client only, (i.e., PowerShell only) the binaries are delivered in a nuget package and the link is not created.

Reply:
I see also above the lines with IsRedHatFamily for libssl.so.10.
But I still don't understand how such fixes work if today we create _single package_ for all Linux.

Reply:
@iSazonov : We don't create a single package for all Linux; we create multiple packages.  This symlink is created at build time for any Ubuntu release but, in practice it is only required for 16.04. If we merge all Ubuntu into a single package, the logic will continue to work.
If we do somehow figure out how to create a single package for all Linux, this model will not work; we'll need to patch at post install time.  From what I've seen to date; that doesn't appear viable in the near term.

Reply:
Thanks for clarify.
Closed.